pkg/docker/index.html

```diff
@@ -152,7 +152,10 @@ <h1 translatable="yes">Can&rsquo;t connect to Docker</h1>
           </tr>
           <tr>
             <td translate>Image:</td>
-            <td colspan="3" id="container-details-image"></td>
+            <td colspan="3">
+            <div id="container-details-image"></div>
+            <div id="container-details-image-id"></div>
```

Comment:
This isn't that same page. This is the detail page only, but yes it will always show a hash and only the name when available. The listing page will show the name if available and only show the hash when it's not available.